### ISSUE TITLE
Toll the Dead - No damage on save

### ISSUE THREAD

**Issue by tposney** [tposney/midi-qol#95]
In GitLab by @will.frame on Oct 12, 2020, 01:45

When my players attack with Sacred flame, the module properly denotes that there is no damage on save and applies 0 damage. However, with Toll the dead, it doesn't seem to know that this is the case and applies damage regardless. The "no damage on save" banner is also missing from the output chat card.

Is there some way to flag the spell so that Midi-QoL recognizes that there is no damage on a save?

**Comment by tposney** [comment]
The easiest is to edit the spell description and put the exact text
No damage on save
in the description. Then when it is cast a save should do no damage.

**Comment by tposney** [comment]
In GitLab by @will.frame on Oct 12, 2020, 02:15

I actually figured out what was going on. It was similar to the issue I had before with weapons that were named the same. I had a regular "Toll the Dead" and a "Toll the Dead D12". Once I renamed the second to "D12 Toll the Dead", the original "Toll the Dead" works correctly, though the second still does not work.

Which brings me to a larger issue I've been having: When a spell/weapon has Versatile damage and you have Speed Rolls on, is there any way to roll the damage set you want, rather than breaking the weapon/spell into two different items as I'm currently doing? Just wondering if I'm doing something incorrectly.

Edit: But adding "no damage on save" in the beginning of the D12 version of the spell did fix the issue for me. Thanks for that. Your module work on Foundry makes my games SO much more pleasurable to run.
